closed/adds/jdk/src/share/classes/sun/security/provider/NativeDigest.java

```diff
@@ -49,13 +50,27 @@
     // length of the message digest in bytes
     private final int digestLength;
     private final int algIndx;
+    private Cleaner cleaner;
```

Comment:
Please remove this field: a reference to the `Cleaner` instance is not needed here.

Reply:
done